assets/css/view.css

```diff
@@ -35,7 +35,7 @@ body {
 }
 /*Dropdown Button */
 .dropbtn {
-	background-color: #b295ba;
+	background-color: #322046;
```

Comment:
I'd prefer to keep the higher contrast button color Jillian had designed.  Can you revert back to #b295ba for this?